### PR TITLE
#1733 CGContextClearRect: Support to clear rect if clipping is not set

### PR DESCRIPTION
**Pull request by msft-Jeyaram**
The current functionality is not implemented properly.
CGContextClearRectwill end up putting the context in an invalid state and any subsequent CGContext[d2d] calls will end up failing. Any application (some of our samples) that uses this call on application init will end up crashing due to failure.

This will unblock our merge to develop, tested with 'sample' applications. They start up and function as expected.

---
This change is [Reviewable](https://reviewable.io/reviews/microsoft/winobjc/1743)

### CODE REVIEW

File: Frameworks/CoreGraphics/CGContext.mm [hunk 1]
```diff
@@ -2068,14 +2068,19 @@ void CGContextShowGlyphsWithAdvances(CGContextRef context, const CGGlyph* glyphs
 
 #pragma region Drawing Operations - Basic Shapes
 /**
- @Status Interoperable
+ @Status Caveat
+ @Notes only supports the scenario where clipping is not set.
```
Comment (DHowett-MSFT) [inline Frameworks/CoreGraphics/CGContext.mm]: Also,
* Only works on untransformed coordinates. The cleared region will be cleared in device space.

File: Frameworks/CoreGraphics/CGContext.mm~RF70a69f.TMP [hunk 2]
```diff
@@ -0,0 +1,2922 @@
+//******************************************************************************
```
Comment (DHowett-MSFT) [inline Frameworks/CoreGraphics/CGContext.mm~RF70a69f.TMP]: ditch this file

Comment (aballway) [inline Frameworks/CoreGraphics/CGContext.mm~RF70a69f.TMP]: remove this file

Comment (msft-Jeyaram) [inline Frameworks/CoreGraphics/CGContext.mm~RF70a69f.TMP]: uhhhh I do not like TMP files

**Comment by MSFTFox** [comment]
:shipit:

**Pull request merged**